src/style.css

```diff
@@ -1,3 +1,73 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+
+  --blue: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.logo {
+  width: 100%;
```

Comment:
add fixed width and height for logo, or max-height and max-weight, maybe you will get too big logo and your header will look broken

Reply:
fix it please :)